electrode_models/air_electrode.py

```diff
@@ -49,17 +49,24 @@ def __init__(self, input_file, inputs, sep_inputs, counter_inputs,
         else:
             raise ValueError("Electrode must be an anode or a cathode.")
 
-        # Store the species index of the Li ion in the Cantera object for the 
+        # Store the species index of the Li ion in the Cantera object for the
         # electrolyte phase:
         self.index_Li_elyte = self.elyte_obj.species_index(inputs['mobile-ion'])
 
         # Phase volume fractions
-        self.eps_host = inputs['eps_host']
-        self.eps_product_init =inputs['eps_product']
-        self.eps_elyte_init = 1 - self.eps_host - self.eps_product_init
+        eps_host = inputs['eps_host']
+        if len(eps_host) == 1:
+            self.eps_host = np.repeat(eps_host[0], self.n_points)
+        elif len(eps_host) == self.n_points:
+            self.eps_host = np.asarray(eps_host)
+        else:
+            raise ValueError("Porosity must be either a single scalar, or must match the discretization of the air electrode.")
```

Comment:
Very small fix: please keep lines within the length limit.  Split to a new line after 'must'.